### PR TITLE
Update 2019 copy

### PR DESCRIPTION
**Pull request by dreamfall**
- Sust.Dev. award
- Change A6/A6.1 for all awards
- Update D section for all awards

### CODE REVIEW

**Comment by dreamfall** [comment]
[[QAE18IMP] Review and update copy for Sustainable Development award (6)](https://trello.com/c/mVQHgFYg/743-qae18imp-review-and-update-copy-for-sustainable-development-award-6)

**Comment by dreamfall** [comment]
[[QAE18IMP] As an applicant, I want two new questions in section A asking if I have applied before, so that Assessors can easily access this information (6)](https://trello.com/c/RdNRNUcL/761-qae18imp-as-an-applicant-i-want-two-new-questions-in-section-a-asking-if-i-have-applied-before-so-that-assessors-can-easily-acce)

File: app/assets/javascripts/frontend/custom_questions/options_with_preselected_conditions_question.js.coffee [hunk 1]
```diff
@@ -24,36 +26,44 @@ window.OptionsWithPreselectedConditionsQuestion = init: ->
...
     # Show C1 question based on this condition
-    if trade_award_and_not_lowest_year
+    if trade_award_and_above_threshold_year
       # Show the correct content
       dependable_children_div_block.find("h2 [data-preselected-condition]").removeClass "display-none"
       dependable_children_div_block.find("h2 [data-preselected-condition='default']").addClass "display-none"
       dependable_controls.addClass "display-none"
 
       # And set selected value for related control
+      console.log dependable_children_div_block
```
Comment (dreamfall) [inline app/assets/javascripts/frontend/custom_questions/options_with_preselected_conditions_question.js.coffee]: ✂️

File: app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb [hunk 2]
```diff
@@ -71,13 +71,15 @@ def development_step2
...
-        textarea :development_desc_long, "Summarise your product, service or management approach" do
+        textarea :development_desc_long, "Summarise your product, service or management approach specifically in relation to your core business and any practices followed which are as a result of the operations or requirements of the wider organisation / parent company. <br/ > <br/ > Please note: If you are applying for a management approach only, the sustainability of your product/service will also be taken into account.
+" do
           classes "word-max-strict"
           ref "B 1.2"
           required
           context %(
             <p>
               If more than one, please make sure you cover all of them. Include a brief description of its/their origin and development.
+
```
Comment (dreamfall) [inline app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb]: ✂️

File: app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb [hunk 3]
```diff
@@ -174,32 +176,88 @@ def development_step2
...
+            </p>
+            <ul>
+              <li>
+                The Government’s goals in the 25 Year Environment Plan to improve the environment within a generation.
+              </li>
+              <li>
+                The goal of demonstrating leadership on cleaner economic growth, as set out in the Government’s Clean Growth Strategy and Industrial Strategy.
+              </li>
+            </ul>
+
```
Comment (dreamfall) [inline app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb]: ✂️

File: app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb [hunk 4]
```diff
@@ -174,32 +176,88 @@ def development_step2
...
+                Reduced waste and increased recycling.
+              </li>
+              <li>
+                Management of exposure to chemicals.
+              </li>
+              <li>
+                Improvements to biosecurity.
+              </li>
+            </ul>
+
```
Comment (dreamfall) [inline app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb]: ✂️

File: app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb [hunk 5]
```diff
@@ -174,32 +176,88 @@ def development_step2
...
+            <p>
+              For example, you might wish to demonstrate how you and your suppliers have:
+            </p>
+
```
Comment (dreamfall) [inline app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb]: ✂️

File: app/assets/javascripts/frontend/custom_questions/options_with_preselected_conditions_question.js.coffee (hunk 1 above)
Comment (rusllonrails) [inline app/assets/javascripts/frontend/custom_questions/options_with_preselected_conditions_question.js.coffee]: Can remove `console.log dependable_children_div_block` block

File: app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb [hunk 6]
```diff
@@ -209,26 +267,62 @@ def development_step2
...
+              \u2022 Substituted material inputs for more environmentally sound alternatives.
+
+              \u2022 Reduced water consumption.
+
+              \u2022 Taken action to encourage customers or the public to reduce their water, energy or other consumption habits, for example by using less plastic.
+
+              \u2022 Protected or enhance the physical environment and its biodiversity on sites that you occupy, those of your supply chain or in your local community.
+
+              \u2022 Participated in external activities with environmental benefits, e.g. supporting local environmental groups, influencing your sector.
+              \u2022 Undertaken sustainable, for example green, procurement.
```
Comment (dreamfall) [inline app/forms/award_years/v2019/sustainable_development/sustainable_development_step2.rb]: Add newline

File: spec/mailers/account_mailers/buckingham_palace_invite_mailer_spec.rb [hunk 7]
```diff
@@ -16,14 +17,22 @@
       kind: "buckingham_palace_reception_attendee_information_due_by"
     ).first
     attendees_info_due.update_column(:trigger_at,
-      DateTime.new(Date.current.year, 5, 6, 00, 00)
+      DateTime.new(
+Date.current.year, 5, 6, 00, 00)
```
Comment (dreamfall) [inline spec/mailers/account_mailers/buckingham_palace_invite_mailer_spec.rb]: Indentation

**Pull request merged**